RetroBar/RetroBar.csproj

```diff
@@ -14,6 +14,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Include="Resources\**">
```

Comment:
This should probably only be png files, rather than all resources.

Reply:
I also thought it would be for png files, but in PR #169 I'm going to add some ico files that are going to be outside the program...

Reply:
I have checked that it copies to the folder and embeds in the executable only the resources it should.